web/html/src/components/datetime/DateTimePicker.tsx

```diff
@@ -155,14 +155,11 @@ class TimePicker extends React.PureComponent<TimePickerProps> {
     this._input?.timepicker({
       roundingFunction: (seconds, options) => seconds,
     });
-    this._input?.on("change", () => {
-      // Do nothing
-    });
     this._input?.on("timeFormatError", () => {
       // Do nothing
     });
     this._input?.timepicker("setTime", this.toFauxBrowserDate(this.props));
-    this._input?.on("changeTime", () => {
+    this._input?.on("changeTime" && "change", () => {
```

Comment:
This looks a bit off, `"changeTime" && "change"` evaluates to `"change"` in Javascript. If only listening to the change event is correct, then it should be `"change"`, if we want to listen to both events then if my jQuery memory doesn't fail me it should be `"changeTime change"`. I'm not sure which one is correct though, if the events overlap then we probably don't want to listen to both. If you tested this code and it worked then we probably only want the change event.

Reply:
Well I know it look weird, but I tried this out and just putting `"change"` won't trigger when you type something in and hit enter. (But thinking about this that's going to be fine, because the change would trigger once you deselect the text field anyway).
Long story short I am going to change it to just `"change"`

Reply:
Updated and tested with just `"change"`